src/trace-agent.js

```diff
@@ -35,8 +36,10 @@ var traceAgent;
 function TraceAgent(config, logger) {
   this.config_ = config;
   this.logger = logger;
+  this.plugins = config && config.plugins ? config.plugins : {};
```

Comment:
@ofrobots I'll keep plugins a part of `config`. The only place that uses the field `this.plugins` is in the plugin loader itself, so I'll make a change there.